src/ext/GeometryUtil.js

```diff
@@ -55,55 +82,67 @@ L.GeometryUtil = L.extend(L.GeometryUtil || {}, {
 	// Converts a metric distance to one of [ feet, nauticalMile, metric or yards ] string
 	//
 	// @alternative
-	// @method readableDistance(distance, isMetric, useFeet, isNauticalMile): string
+	// @method readableDistance(distance, isMetric, useFeet, isNauticalMile, precision): string
 	// Converts metric distance to distance string.
-	readableDistance: function (distance, isMetric, isFeet, isNauticalMile) {
+	// The value will be rounded as defined by the precision option object.
+	readableDistance: function (distance, isMetric, isFeet, isNauticalMile, precision) {
 		var distanceStr,
-			units;
+			units,
+			precision = L.Util.extend({}, defaultPrecision, precision);
 
-		if (typeof isMetric == "string") {
-			units = isMetric;
+		if (isMetric) {
+			units = typeof isMetric == 'string' ? isMetric : 'metric';
```

Comment:
Oh I need to resolve the merge conflict, but that shouldn't be any problem.

@roberrrt-s Yeah, look at the tests at https://github.com/Leaflet/Leaflet.draw/pull/672/files#diff-5f040072bde7f398b0be398b6e435915R20 I think that's what you mean?